picoss/yousign-bundle/2.2/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Picoss\\YousignBundle\\PicossYousignBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "#1": "To create a demo account, visit https://demo.yousign.fr/fr/public/api/registration",
+        "#2": "YOUSIGN_ENV should be 'demo' or 'prod'",
```

Comment:
I don’t understand this comment.

Reply:
I've removed the comments, and put them in the post-install.txt